packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -119,19 +121,52 @@ const getParticipantsForDefaultRender = (
  * assigned then each participant is rendered with `ParticipantItem`.
  */
 export const ParticipantList = (props: ParticipantListProps): JSX.Element => {
-  const { excludeMe = false, myUserId, participants, onParticipantRemove, onRenderAvatar, onRenderParticipant } = props;
+  const {
+    excludeMe = false,
+    myUserId,
+    participants,
+    onParticipantRemove,
+    onRenderAvatar,
+    onRenderParticipant,
+    onFetchParticipantMenuItems
+  } = props;
+
   const ids = useIdentifiers();
+  const locale = useLocale().strings.participantItem;
```

Comment:
Nit. Would change this variable name to `const strings = useLocale().strings.participantItem;`. But otherwise, localization is good.